layouts/product/iac.html

```diff
@@ -0,0 +1,286 @@
+{{ define "hero" }}
+    {{ partial "hero" (dict "title" .Params.heading) }}
+{{ end }}
+
+{{ define "main" }}
+    <section id="overview" class="my-8 relative">
+        <div class="shape-background product-overview-section">
```

Comment:
we should remove those obnoxious red bubbles

Reply:
@shughes26 Can you help with this?